Added release note for R34 (#792)
pxblue/doc-it: src/docs/release-notes/R34/FullNotes.mdx

import React from 'react';

## React User Managment Workflows

-   Developed a user management workflow for roles and operations in React, workflow includes functionalities for adding, editing, viewing, and deleting user roles and operations.

> -   This functionality will be available in future release once all the user management workflow architecture are complete.

Reviewer: ```suggestion
-   This feature will be available in a future release once the entire user management workflow architecture is complete.
```

Author: Closing this PR due to some issue in branch.
New PR : https://github.com/etn-ccis/blui-doc-it/pull/793

## React

-   Updated packages to support MUI v6.

## Package Summary

### React

-   @brightlayer-ui/react-components v7.0.0 ([Changelog](https://github.com/etn-ccis/blui-react-component-library/blob/master/CHANGELOG.md))
-   @brightlayer-ui/react-themes v8.0.0 ([Changelog](https://github.com/etn-ccis/blui-react-themes/blob/master/CHANGELOG.md))
-   @brightlayer-ui/react-auth-workflow v6.0.0 ([Changelog](https://www.npmjs.com/package/@brightlayer-ui/react-auth-workflow))

### React Native

> -   @brightlayer-ui/react-native-auth-workflow v7.0.1 ([Changelog](https://www.npmjs.com/package/@brightlayer-ui/react-native-auth-workflow))

Author: Updated links, Thanks
